immersion/kubeflow_pipelines/pipelines/solutions/lab-02_vertex.ipynb

```diff
@@ -0,0 +1,444 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    ARTIFACT_STORE = f'gs://{PROJECT_ID}-vertex'
</pre><p>We assume that this bucket exists, similar to other labs we should create this if does not exist, or call it out in text.</p><p> </p><pre class="ql-syntax" spellcheck="false">!gsutil ls | grep ^{ARTIFACT_STORE}/$ || gsutil mb -l {REGION} {ARTIFACT_STORE}
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/mlops-on-gcp/pull/116/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>Added that. Thanks!</p>